modules/ngx_http_upstream_dyups_module/ngx_http_dyups_module.c

```diff
@@ -745,6 +756,13 @@ ngx_dyups_delete_upstream(ngx_str_t *name, ngx_str_t *rv)
         status = NGX_HTTP_INTERNAL_SERVER_ERROR;
```

Comment:
should added `goto finish`

Reply:
thanks for your remind, and I have added it.